### DISCUSSION TITLE
Searching for a new maintainer

### DISCUSSION THREAD

**Discussion by IJustDev** [IJustDev/Gitea-VSCode#50] [General]
Hey everybody,

I created this plugin when I was using Gitea for a side project, back when I was in high school.
Unfortunately the last time I have used Gitea is a long time ago and due to my other projects I do not often find the time to do something for this project.

Since at least a few people really use it on a daily basis and create pull requests I do not want to let you wait months.

So I would love to find somebody to maintain this project:
- reviewing PRs
- testing PRs
- merging PRs
- releasing the app.

If anybody who - knows the project well - wants to do this job and enhance the project, just respond to this conversation.

**Comment by MaxenceG2M** [comment]
Hello IJustDev,

I'm in :)

**Reply by IJustDev** [reply]
You look very suitable and did a great job extending functionality of this extension. Check your emails!
